packages/web-runtime/package.json

```diff
@@ -30,7 +30,7 @@
     "vue-resize": "^0.4.5",
     "vue-router": "^3.1.3",
     "vue-scrollto": "^2.15.0",
-    "vue-select": "^3.11.2",
+    "vue-select": "^3.12.0",
```

Comment:
this also needs to applied in ODS

Reply:
https://github.com/owncloud/owncloud-design-system/pull/1536